src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -56,7 +55,9 @@
     <Compile Include="System\Data\ConstraintCollection.cs" />
     <Compile Include="System\Data\ConstraintConverter.cs" />
     <Compile Include="System\Data\ConstraintEnumerator.cs" />
-    <Compile Include="System\Data\DataColumn.cs" />
+    <Compile Include="System\Data\DataColumn.cs">
+      <SubType>Component</SubType>
+    </Compile>
```

Comment:
Looks like VS may have made some edits here?  Can you remove the ones like this that weren't intentional?